src/source/ObjcppMarshal.scala

```diff
@@ -49,7 +47,6 @@ class ObjcppMarshal(spec: Spec) extends Marshal(spec) {
 
   def include(m: Meta) = m match {
     case d: MDef => d.defType match {
```

Comment:
Looks like this `match` here is no longer needed, you can simply return the result here.